src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsPullsListener.php

```diff
@@ -231,6 +244,12 @@ class JoomlacmsPullsListener extends AbstractListener
 			'^administrator/components/com_fields',
 			'^components/com_fields',
 		],
+		// Composer Change
+		'72' => [
```

Comment:
And than we add another string => id conversion to get the category id? Please send me a complete PR with your suggestions. Thanks.